gpu_direct_rdma_access.c

```diff
@@ -448,6 +448,50 @@ static int modify_source_qp_to_rtr_and_rts(struct rdma_device *rdma_dev)
     return 0;
 }
 
+static int close_qp(struct ibv_qp *qp) 
+{
+    int ret;
+    if (qp) {
+        DEBUG_LOG("ibv_destroy_qp(%p)\n", qp);
+        ret = ibv_destroy_qp(qp);
+    }
+    return ret;
+}
+
+static int setup_server_qp_state(struct rdma_device *rdma_dev) 
```

Comment:
current name is to ambiguous
maybe use a more specific API name: modify_source_qp_rst2rts()